Makefile

```diff
@@ -193,6 +193,10 @@ ${RELEASE_TOOL}:
 ${GOLANGCI_LINT}:
 	$(call go-build,golangci-lint,./vendor/github.com/golangci/golangci-lint/cmd/golangci-lint)
 
+vendor-update:
```

Comment:
I don't feel strongly about it, but I prefer to force all developers actually using `go` for that (mostly for educational purposes). Adding an alias for `go get -u` seems a bit redundant.

Reply:
Hm, I think I will drop it and update the CONTRIBUTING.md guideline.